status-im/status-react
# adding ugly delivery states for chat

Delivery states are as follows :
- Sending (all messages begin with sending state)
- Sent (They are sent if they have left the device)
- Received (They are received once the receiver has acknowledged the message)

fixes #14588

#### Platforms
- Android
- iOS

##### Functional
- 1-1 chats
- public chats
- group chats

### Steps to test
- Open Status
- Open Chat
- Send a message and see if you can see the first delivery state "Sending" which is a single dot( it will quickly switch to second state depending upon your network connection and peers.
- The second state you see would be two dots which indicate "Sent"
- The third state would be three dots which indicates "Received". This means that the person you sent the chat to has received it.

### screen recording 
https://user-images.githubusercontent.com/64726664/209079234-b44a6d14-f228-45e3-b31b-254a53b64b08.mov


status: ready

ps : I've been told to make this red & ugly :)
